src/components/MainParticipant/MainParticipant.test.tsx

```diff
@@ -5,16 +5,16 @@ import { shallow } from 'enzyme';
 import { mockLocalParticipant, mockParticipant } from '../../utils/mocks';
 import rootStore from '../../stores';
 
-const localParticipant = new mockLocalParticipant();
+const localParticipant = new mockLocalParticipant('local@Reporter@1');
```

Comment:
```suggestion
const localParticipant = new mockLocalParticipant('local@Reporter@1');
```
I prefer three parameters, which you build the string with @ internally